src/main/resources/develocity/gradle/develocity-init-script.gradle

```diff
@@ -169,6 +200,12 @@ if (GradleVersion.current() < GradleVersion.version('6.0')) {
                             buildScan.allowUntrustedServer = develocityAllowUntrustedServer
                         }
                     }
+
+                    if (buildScanTermsOfUseUrl && buildScanTermsOfUseAgree) {
+                        buildScan.termsOfServiceUrl = buildScanTermsOfUseUrl
+                        buildScan.termsOfServiceAgree = buildScanTermsOfUseAgree
+                    }
```

Comment:
We don't pass these values from the Bamboo plugin configuration, so why have them here?

Reply:
These are required for GitHub Actions, and possibly could be useful for other CI integrations. If the Bamboo plugin doesn't configure the inputs, then this will have no impact.

In order to have a shared reference implementation of the init-script, we can either:
- Leave these in the Bamboo init-script as a no-op
- Implement a build process in the Bamboo plugin to selectively remove
- Implement a build process in the GitHub Action to selectively add this feature

In the end, I think it's simplest to have a single common init-script with a superset of required features.

Reply:
Yup, same comment as I made above about consistency, let's keep it